gather-rpms.py

```diff
@@ -16,7 +16,14 @@
 CG_IMPORT_JSON = "cg_import.json"
 SBOM_JSON = "sbom-spdx.json"
 NVR_FILE = "nvr.log"
-# TODO - repo_id, maybe whole mock-config
+
+# JSON with extra metadata
+# repo_id: koji build repo id
+# buildroot_tag: koji build tag
+# event_id: koji event id of build repo creation
+# {"repo_id": <repo_id>, "buildroot_tag": <buildroot_tag>, "event_id": <event_id>}
```

Comment:
Suggestion for wording:

This is a metadata file that can be offered by RHEL/Fedora pipeline flavors that employ Koji buildroots. The file provides info related to the utilized Koji buildroot.  The expected file format is JSON with key-value pairs:
repo_id: ...